db/db-challenge3/db-010/0010.sql

```diff
@@ -0,0 +1,14 @@
+SELECT c.name, d.`最新投稿日`, u.name
+FROM (SELECT p.chat_room_id AS `チャットルームID`, MAX(p.posted_at) AS `最新投稿日` 
```

Comment:
<修正不要>
join目的のサブクエリのカラム名に日本語を使うことは少ない気がします。（計測範囲は、僕周辺ですが）。
カラム名に日本語を使うのは、最終的に表示されるカラム名を変更するときくらいですかね。

日本語（というか、マルチバイト）を避ける理由としては、下記です。
- 思わぬバグが入ってしまうことがある（文字コード周りは闇が深いこと多いので、経験的に避けていることが多い）
- 都度「`」を使って文字を囲むのがめんどくさい
- みんな日本語が読めるとは限らない（日本人じゃない人と働くかも！）

最後のは、シチュエーションとしてはありうりますが、僕は体験したことがありませんが…


```suggestion
FROM (SELECT p.chat_room_id AS chat_room_id, MAX(p.posted_at) AS latest_posted_at 
```

Reply:
承知いたしました！
いろいろリスクは避けなければならないですね。
英語名で記述するようにします！
レビューありがとうございます！